contentcuration/contentcuration/frontend/channelEdit/vuex/assessmentItem/actions.js

```diff
@@ -19,23 +27,40 @@ export function loadAssessmentItem(context, id) {
     });
 }
 
-/* ASSESSMENTITEM EDITOR ACTIONS */
-export function updateAssessmentItems(context, assessmentItems) {
-  if (!Array.isArray(assessmentItems)) {
-    throw TypeError('assessmentItems must be an array of assessmentItems');
-  }
-  // Keep all assessment items in vuex state
-  context.commit('ADD_ASSESSMENTITEMS', assessmentItems);
-  // Only commit assessment items that pass sanitization
-  return Promise.all(
-    sanitizeAssessmentItems(assessmentItems).map(assessmentItem =>
-      AssessmentItem.put(assessmentItem)
-    )
-  );
+export function addAssessmentItem(context, assessmentItem) {
+  // API accepts answers and hints as strings
+  const stringifiedAssessmentItem = {
+    ...assessmentItem,
+    answers: JSON.stringify(assessmentItem.answers || []),
```

Comment:
Hrm, should we update this in the API to avoid this?

Reply:
https://github.com/learningequality/studio/issues/1855